.github/workflows/tf_plan_apply.yml

```diff
@@ -0,0 +1,40 @@
+name: Manual - Terraform Plan / Apply
+
+on:
+  workflow_dispatch:
+    inputs:
+      action_type:
+        type: choice
+        description: What actions should be done?
+        required: true
+        default: Plan & Apply
+        options: 
+        - Plan
+        - Apply
+        - Plan & Apply
+
+jobs:
+    plan_apply:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout repository
+          uses: actions/checkout@v2
+
+        - name: Log in with Azure
+          uses: azure/login@v1
+          with:
+            creds: '${{ secrets.AZURE_CREDENTIALS }}'
+
+        - name: Install terraform
+          uses: hashicorp/setup-terraform@v1
+
+        - name: Download terraform dependencies
+          run: terraform init
+            
+        - name: Terraform Plan
+          if: contains(fromJSON('["Plan", "Plan & Apply"]'), github.event.inputs.action_type)
+          run: terraform plan -input=false
```

Comment:
Pentru ce e nevoie de -input=false aici?

Reply:
In momentul in care se da plan variabilele folosite si nedeclarate / nedetectate in environment o sa aiba un prompt in CLI-ul de la terraform. Am zis sa il folosesc pentru ca:
1. E default la template-ul de workflow de la HashiCorp
2. Mi se pare mai ok sa dea workflow-ul eroare in loc sa stea in asteptare si sa nu primeasca input pana eventual isi ia timeout.

Reply:
Ok, are sens